tests/unit/schema/test_api.py

```diff
@@ -2737,6 +2737,10 @@ def test_read_empty_python_settings_from_service_yaml():
     api_schema = api.API.build(fd, "google.example.v1beta1", opts=cli_options)
     assert api_schema.all_library_settings["google.example.v1beta1"].python_settings \
         == client_pb2.PythonSettings()
+    assert api_schema.all_library_settings["google.example.v1beta1"].python_settings.experimental_features \
+        == client_pb2.PythonSettings.ExperimentalFeatures()
```

Comment:
Can you link me to this property in `client_pb2` file?

Reply:
https://github.com/googleapis/googleapis/blob/6a474b31c53cc1797710206824a17b364a835d2d/google/api/client.proto#L281

https://github.com/googleapis/python-api-common-protos/blob/f30115b7302afc91bf06f210a270a6530b7fafdf/google/api/client.proto#L281

https://github.com/googleapis/python-api-common-protos/pull/249